drivers/Kconfig

```diff
@@ -24,6 +24,22 @@ config DRVR_MKRD
 		the selecting this option will also enable the BOARDIOC_MKRD
 		command that will support creation of RAM disks from applications.
 
+# ARCH needs to support memory access while CPU is running to be able to use
+# the LWL CONSOLE
+
+config ARCH_HAVE_RDWR_MEM_CPU_RUN
```

Comment:
should we remove arm version?
https://github.com/apache/incubator-nuttx/blob/master/arch/arm/src/common/arm_lwl_console.c

Reply:
Yes, it is a separated commit, I don't want to mix the generic driver with arch specific code/modifications. Let's focus in the generic LWL driver, the next commit will remove the arch driver, enable the ARCH_HAVE_RDWR_MEM_CPU_RUN and replace CONFIG_ARM_LWL_CONSOLE with CONFIG_LWL_CONSOLE.